Clarify Window.Activate foreground behavior (#125)
MicrosoftDocs/winapps-winrt-api: microsoft.ui.xaml/window_activate_1797342875.md

## -description

> Attempts to activate the application window. The window must be attached to the calling thread's message queue. This method doesn't force a background app into the foreground. If the app is in the foreground when the system activates the window, the window is brought to the foreground (top of the Z-order) and receives input focus.

Reviewer: @JesseCol friendly nudge on this one — the intent (fixing the overclaim about unconditional foreground activation) is right and worth landing. The blocker is that the wording pulls in Win32 `SetActiveWindow` semantics that don't fit the WinUI 3 abstraction:

- **"must be attached to the calling thread's message queue"** — a `Microsoft.UI.Xaml.Window` is already bound to its UI/dispatcher thread. A wrong-thread call throws a thread-affinity exception; it doesn't silently no-op the way this clause implies. This is a Win32 implementation detail that shouldn't surface here.
- **"when the system activates the window"** reads circularly inside a description of what this method itself does.

Here's a replacement that keeps the valuable foreground clarification without the Win32 leakage — you can accept it directly:

```suggestion
Attempts to activate the application window. If the calling app is in the foreground, the window is brought to the foreground (top of Z-order) and receives input focus. If the app is in the background, this method does not force the window into the foreground.
```

Author: OK to say in the docs that under the hood WinUI is calling SetActiveWindow()?  Then the reader could follow that link to read more.
